### PR TITLE
update ganache-core to v2.13.0-beta.0

### PR DESCRIPTION
**Pull request by davidmurdoch**

### CODE REVIEW

**Comment by dragonsky7** [comment]
Mabe a amplification of deutérium of 2/2.5 kilo Watt RMS ajust a ion carbon to the circonférence.
Jean-Marc Thiffault

**Review by melyalex — APPROVED** [review]

**Review by dragonsky7 — APPROVED** [review]
I am sure that it need a similar injection on phase whit the deutérium base the injection got to have lithium-ion whit brass recipient. Jean-Marc Thiffault

**Comment by dragonsky7** [comment]
The injection got to work whit brass recipient and the ignition provide whit 2/2.1 proton fusion stabilisator.
Tell me if a am in the good way.

**Review by melyalex — APPROVED** [review]

File: packages/db/src/artifacts/test/index.ts [hunk 1]
```diff
@@ -247,4 +241,15 @@ describe("Artifacts queries", () => {
...
+  it("cleanup", done => {
+    // smething in the tests above starts a doing, uh, something and causes jest
+    // to loose its cool. I just want my PR to pass so I can release
+    // ganache-core to @latest so i'm committing this little guy so jest doesn't
+    // wig out.
+    setTimeout(() => {
+      expect(1).toEqual(1);
+      done();
+    }, 4000);
+  });
```
Comment (davidmurdoch) [inline packages/db/src/artifacts/test/index.ts:254]: we should talk about this.

**Review by davidmurdoch — COMMENTED** [review]
I don't intend tfor this to be merge, since this is an update to a beta release of ganache-core... I just want to make sure this next ganache-core release will be compatible with truffle. That's what this PR is for.

I had two issues:

1. jest: it didn't like node Buffers so i had to update it and ts-jest
2. jest: one of the test files does stuff _after_ the tests are complete. jest freaks out and proceeds to break unrelated tests because of it. Jest just has has zero chill.

I hope you enjoyed this comment. Have a nice day.

**Comment by fainashalts** [comment]
Closing this PR in favor of this one: https://github.com/trufflesuite/truffle/pull/3435/

**Pull request closed without merging**